src/sections/support-page/support-channels/support-channels.module.scss

```diff
@@ -0,0 +1,68 @@
+@import '~@bit/bit.base-ui.layout.breakpoints';
+
+.wrapper {
+    margin-bottom: 30px;
+}
+
+
+.mainGrid {
+	gap: 20px;
+    align-items: center;
+    margin-bottom: 150px;
+}
+
+.image {
+    max-width: 100%;
+    margin: 0 auto;
+    display: none;
+    @media screen and (min-width: $br-md) {
+        display: unset;
+    }
+}
+
+.content {
+    margin: 0 auto;
+}
+
+.mainParagraph {
+    margin-bottom: 30px;
+    color: var(--text-muted, #6c707c);
+}
+.link {
+    text-decoration: none;
+    display: flex;
+    justify-content: center;
+    width: fit-content;
+    margin: auto;
+    @media screen and (min-width: $br-md) {
+        margin: unset;
+        justify-content: flex-start;
+    }
+}
+
+.paragraph {
+    margin-bottom: 15px;
+    color: var(--text-muted, #6c707c);
+}
+
+.button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: var(--brand-text, white);
+    height: 38px;
+    width: 180px;
+
+    > span {
+        margin-right: 5px;
+    }
+}
+.slack {
+    background-color: var(--slack-bg, #4a154b);
+}
+.email {
+    background-color: #6c5ce7;
```

Comment:
this is $p50 / --base-color / cta button

Reply:
updated